Cql/CodeGeneration.NET/Visitors/SimplifyExpressionsVisitor.cs

```diff
@@ -152,26 +162,30 @@ protected override Expression VisitBinary(BinaryExpression node)
 
                 // The interim value of an assignment is clear, we don't need to simplify
                 { NodeType: ExpressionType.Assign } => base.VisitBinary(node),
-
-                _ => simplify(base.VisitBinary(node))
+                
```

Comment:
Remove whitespace-only line

Reply:
All the cases have a whitespace-only line here, so make it clear to which line the comment belongs....

Reply:
In the code diff it's a bit clearer on which line the comment was placed and also highlighting the whitespace. It was on line 165

Reply:
Weird....that must be a diff artifact!  I would not want an empty line AFTER that default return either...

![image](https://github.com/FirelyTeam/firely-cql-sdk/assets/2276305/57a2e815-8506-4522-839a-87cbcc54625a)

Reply:
The whitespace that I'm referring to is on the line just above `_ => MakeLet...`. That line contains only spaces. Could you add the following line to editorconfig as well? `trim_trailing_whitespace = true`

Reply:
Ah, I cannot see that.  Yes, I'll add that to the .editorconfig

Reply:
I've added it, but this whitespace is invisible to me, and very easy to introduce: if I press enter after a line, the editor will insert whitespace so my cursor aligns with the previous line (which makes sense), but that whitespace will stay there.